### PR TITLE
TiFlash: Add FAQ for TiFlash trouble-shooting when failed to start

### PR DESCRIPTION
**Pull request by birdstorm**
### What is changed, added or deleted? (Required)

Add FAQ for TiFlash trouble-shooting when failed to start

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### CODE REVIEW

File: tiflash/troubleshoot-tiflash.md [hunk 1]
```diff
@@ -8,6 +8,32 @@ category: reference
...
+
+2. 检查系统的 `ulimit` 参数设置。
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    ulimit -n 1000000
+    ```
+
+3. 使用 PD CTL，检查是否有之前未成功下线的 TiFlash 节点，并将它们强制下线。（参考[手动缩容 TiFlash 节点](/scale-tidb-using-tiup.md#方案二手动缩容-tiflash-节点)）
```
Comment (flowbehappy) [inline tiflash/troubleshoot-tiflash.md:33]:
```suggestion
3. 使用 PD CTL，检查在该节点（相同 IP 和 Port）是否有之前未成功下线的 TiFlash 实例，并将它们强制下线。（参考[手动缩容 TiFlash 节点](/scale-tidb-using-tiup.md#方案二手动缩容-tiflash-节点)）
```

Comment (flowbehappy) [reply tiflash/troubleshoot-tiflash.md]: 再加个命令示例吧

File: tiflash/troubleshoot-tiflash.md [hunk 2]
```diff
@@ -8,6 +8,32 @@ category: reference
...
+
+2. 检查系统的 `ulimit` 参数设置。
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    ulimit -n 1000000
+    ```
+
+3. 使用 PD CTL，检查在该节点（相同 IP 和 Port）是否有之前未成功下线的 TiFlash 实例，并将它们强制下线。（参考[手动缩容 TiFlash 节点](/scale-tidb-using-tiup.md#方案二手动缩容-tiflash-节点)）
```
Comment (yikeke) [inline tiflash/troubleshoot-tiflash.md:33]:
```suggestion
3. 使用 PD Control 工具检查在该节点（相同 IP 和 Port）是否有之前未成功下线的 TiFlash 实例，并将它们强制下线。（参考[手动缩容 TiFlash 节点](/scale-tidb-using-tiup.md#方案二手动缩容-tiflash-节点)）
```

File: tiflash/troubleshoot-tiflash.md [hunk 3]
```diff
@@ -8,6 +8,32 @@ category: reference
...
+
+4. 如果遇到上述方法无法解决的问题，可以打包 TiFlash 的 log 文件夹，并在 [AskTUG](http://asktug.com) 社区中提问。
```
Comment (yikeke) [inline tiflash/troubleshoot-tiflash.md:35]:
```suggestion
如果遇到上述方法无法解决的问题，可以打包 TiFlash 的 log 文件夹，并在 [AskTUG](http://asktug.com) 社区中提问。
```

**Comment by birdstorm** [comment]
@yikeke PTAL again, thx!

File: tiflash/troubleshoot-tiflash.md [hunk 4]
```diff
@@ -8,6 +8,32 @@ category: reference
...
+
+2. 检查系统的 `ulimit` 参数设置。
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    ulimit -n 1000000
+    ```
+
+3. 使用 PD Control 工具检查在该节点（相同 IP 和 Port）是否有之前未成功下线的 TiFlash 实例，并将它们强制下线。（参考[手动缩容 TiFlash 节点](/scale-tidb-using-tiup.md#方案二手动缩容-tiflash-节点)）
```
Comment (yikeke) [inline tiflash/troubleshoot-tiflash.md:33]:
```suggestion
3. 使用 PD Control 工具检查在该节点（相同 IP 和 Port）是否有之前未成功下线的 TiFlash 实例，并将它们强制下线。（下线步骤参考[手动缩容 TiFlash 节点](/scale-tidb-using-tiup.md#方案二手动缩容-tiflash-节点)）
```

**Pull request merged**